src/sentry/static/sentry/app/views/settings/project/projectCreateServiceHook.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {WithRouterProps} from 'react-router';
+
+import {t} from 'app/locale';
+import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
+import ServiceHookSettingsForm from 'app/views/settings/project/serviceHookSettingsForm';
+
+type Props = WithRouterProps<{orgId: string; projectId: string}, {}>;
+
+function ProjectCreateServiceHook({params}: Props) {
```

Comment:
I agree that the AsyncView didn't make much sense here, but should we maybe keep the page title?